.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,27 @@
-### 🛠 Purpose
+### Purpose 🚀
+_clear, one sentence description of primary purpose of this PR_
 
----
+###  Notes ✏️
+_details of code change / secondary purposes of this PR_
 
-### ✏️ Notes to Reviewer
+### Updates 📦
+- [ ] MAJOR (breaking) change to _these packages_
+- [ ] MINOR (backward compatible) change to _these packages_ 
+- [ ] PATCH (big fix) change to _these packages_
 
----
+### Storybook 📙
+http://storybooks.highbond-s3.com/paprika/your-branch-name
+
+### Screenshots 📸
+_optional but highly recommended_
+
+### References 🔗
+_Jira ticket / GitHub issues_
 
-### 🖥 Screenshots
 
 ---
+### Resources 🔖
```

Comment:
I wonder if it makes sense to wrap this into `<!--` `-->` so that it's visible in the editor when creating the PR, but not in the created PR? I don't think these are valuable to reviewers.

Reply:
+1